assets/src/dashboard/components/scrollToTop/index.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * External dependencies
+ */
+import { useRef, useReducer, useLayoutEffect } from 'react';
+import styled from 'styled-components';
+import { useDebouncedCallback } from 'use-debounce';
+/**
+ * Internal dependencies
+ */
+import { BEZIER } from '../../constants/animation';
+import { ReactComponent as DropUpArrowSvg } from '../../icons/dropUpArrow.svg';
+
+const ScrollButton = styled.button`
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  height: 50px;
+  width: 50px;
+  visibility: ${({ isVisible }) => (isVisible ? 'visible' : 'hidden')};
+  cursor: pointer;
+  border-radius: 50%;
+  border: 1px solid transparent;
+  box-shadow: 0px 4px 14px rgba(0, 0, 0, 0.25);
+  color: ${({ theme }) => theme.colors.gray900};
+  background-color: ${({ theme }) => theme.colors.white};
+  opacity: ${({ isVisible }) => (isVisible ? 1.0 : 0)};
+  transition: opacity 0.75s ${BEZIER.outSine};
```

Comment:
Okay so super small design nit. Properties like opacity and color transitions are better perceived with a `linear` ease. Properties like `scale` and `translation` benefit more from either physics or some type of smoothing like this `BEZIER.outSine` here. so may be best to just do:
``` 
transition: opacity 0.75s linear;
```

Definitely a super minute detail and nobodies probably gonna notice the difference here, but something to use in the future to keep all your interactions super tight.

I think the creator of popmotion had a small post or blurb about this a while ago in their documentation. lemme see if I can dig up

Reply:
Gahh he used to have all these animation musings in the docs for popmotion but they've since been removed ☹️ 

He mentions in one of his blog posts that he may do an entire post on easing, but looks like it's not somethin he's done yet

Reply:
thank you for pointing this out, i had no idea of that philosophy!  If you find any blog posts on that or have a resource to point me for this I'd love it.